internal/api/list_projects_v1alpha1.go

```diff
@@ -13,7 +14,7 @@ import (
 
 func (s *server) ListProjects(
 	ctx context.Context,
-	_ *connect.Request[svcv1alpha1.ListProjectsRequest],
+	req *connect.Request[svcv1alpha1.ListProjectsRequest],
```

Comment:
The test are failing because they're passing a nil request. You'll have to update them to pass a legitimate request now.